### PR TITLE
support for ETP’s tests

### PR DESCRIPTION
**Pull request by cliffordp**
[ETP-471]

For https://github.com/moderntribe/event-tickets-plus/pull/1089

[ETP-471]: https://moderntribe.atlassian.net/browse/ETP-471

### CODE REVIEW

File: src/views/v2/tickets/item/extra.php [hunk 1]
```diff
@@ -76,13 +76,14 @@
...
 <div <?php tribe_classes( $classes ); ?>>
 
-	<?php $this->template( 'v2/tickets/item/extra/price' ); ?>
+	<?php $this->template( 'v2/tickets/item/extra/price', $context ); ?>
 
-	<?php $this->template( 'v2/tickets/item/extra/available' ); ?>
+	<?php $this->template( 'v2/tickets/item/extra/available', $context ); ?>
 
-	<?php $this->template( 'v2/tickets/item/extra/description-toggle' ); ?>
+	<?php $this->template( 'v2/tickets/item/extra/description-toggle', $context ); ?>
```
Comment (sc0ttkclark) [inline src/views/v2/tickets/item/extra.php:87]: $context should be necessary here

File: src/views/v2/tickets/item/content.php [hunk 2]
```diff
@@ -50,8 +50,16 @@
...
+	[
+		'ticket'  => $ticket,
+		'post_id' => $post_id,
+	]
+);
 
-$this->template( 'v2/tickets/item/extra' );
+$this->template( 'v2/tickets/item/content/description', $context );
+
+$this->template( 'v2/tickets/item/extra', $context );
```
Comment (sc0ttkclark) [inline src/views/v2/tickets/item/content.php:65]: None of these changes should be necessary, if they are, let's save whatever the root cause is for a future PR to address it in common.

File: src/Tribe/Tickets_View.php [hunk 3]
```diff
@@ -962,33 +962,32 @@ protected function normalize_rsvp_option( &$option ) {
 	 *
 	 * @since 4.11.0
 	 * @since 4.12.3 Update usage of get_event_ticket_provider().
+	 * @since TBD Moved getting of context array to its own method.
```
Comment (sc0ttkclark) [inline src/Tribe/Tickets_View.php:965]: This is useful, but it's late in the sprint to make this change, we need to workaround it for now until a future sprint

**Review by sc0ttkclark — CHANGES_REQUESTED** [review]
Some changes here, some stuff to save for a future sprint too

**Comment by juanfra** [comment]
Closing as the linked PR was no longer relevant https://github.com/the-events-calendar/event-tickets-plus/pull/1089#issuecomment-776018732

**Pull request closed without merging**